src/main/resources/application.properties

```diff
@@ -35,10 +35,17 @@ ccd.case.search.wildcards.allowed=false
 ccd.dm.domain=${CCD_DM_DOMAIN:https://api-gateway.dev.dm.reform.hmcts.net}
 
 auth.idam.client.baseUrl=${IDAM_USER_URL:http://localhost:4501}
-auth.provider.service.client.baseUrl=${IDAM_S2S_URL:http://localhost:4502}
-auth.provider.service.client.microservice=ccd_data
-auth.provider.service.client.key=${DATA_STORE_IDAM_KEY:AAAAAAAAAAAAAAAB}
-auth.provider.service.client.tokenTimeToLiveInSeconds=900
+
+# This property *must* be retained because it is required by the ServiceTokenParserConfiguration class in the
+# auth-checker-lib library, on which there is a dependency
+auth.provider.service.client.baseUrl=${IDAM_S2S_URL:http://rpe-service-auth-provider-saat.service.core-compute-saat.internal:4502}
+
+# Required for the ServiceAuthorisationApi class in service-auth-provider-java-client library
```

Comment:
isn't it required by `ServiceTokenGeneratorConfiguration`?

Reply:
No - that class doesn't exist any more in `auth-checker-lib`. See https://github.com/hmcts/auth-checker-lib/blob/master/src/main/java/uk/gov/hmcts/reform/auth/parser/idam/spring/service/token/ServiceTokenParserConfiguration.java#L25